specification/v1/bindings/restful/swagger.yaml

```diff
@@ -60,13 +60,13 @@ paths:
         Verifies if the specified ecosystem (`ecosystem_id`) is recognized under the given governance framework (`egf_id`).
       operationId: checkEcosystemRecognition
       parameters:
-        - name: ecosystem_id
+        - name: entity_id
           in: path
           required: true
           description: Unique identifier of the ecosystem being queried.
           schema:
             type: string
-        - name: egf_id
+        - name: authority_id
           in: query
           required: true
           description: Unique identifier of the governance framework. Defaults to single egf_id of the Trust Registry itself.
```

Comment:
Unique identifier of the ecosystem governing authority asserting recognition. Defaults to the ecosystem governing authority of the trust registry (but only if the trust registry serves only a single ecosystem governing authority).